### PR TITLE
update docs with new config

### PR DESCRIPTION
**Pull request by luluhoc**
@adrien2p updated docs

### CODE REVIEW

**Review by adrien2p — CHANGES_REQUESTED** [review]

File: docs/pages/authentication/auth0.mdx [hunk 1]
```diff
@@ -35,46 +35,33 @@ newly added plugins. To do so here are the steps
...
-                    // expiresIn: 24 * 60 * 60 * 1000,
-                    // verifyCallback: (container, req, accessToken, refreshToken, extraParams, profile, strict) => {
-                    //    // implement your custom verify callback here if you need it
-                    // }
+        options: [{
+                type: "auth0",
+                // strict: "all", // or "none" or "store" or "admin"
+                strict: "none",
+                identifier: "auth0",
+                auth0: {
```
[the remark this answers is not in the document]
Comment (adrien2p) [reply docs/pages/authentication/auth0.mdx:43]: instead, you can directly pass admin/store options at the same level as the identifier/type

**Comment by adrien2p** [comment]
Thanks a lot for your contribution 💪 greatly appreciated

**Review by adrien2p — APPROVED** [review]
LGTM, they might just be some indentation issues

**Comment by luluhoc** [comment]
> LGTM, they might just be some indentation issues

fixed indentation issues etc

**Review by adrien2p — APPROVED** [review]

**Pull request merged**